pkg/virt-handler/vm_test.go

```diff
@@ -1151,6 +1157,53 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(blockMigrate).To(BeTrue())
 			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared HostDisk")))
 		})
+
+		It("should block migrate for empty CPU", func() {
```

Comment:
I'm fine with these tests. :+1:

However, I have two *optional* suggestions:
* you could reduce duplication if you'd convert these tests into a `Table`, maybe adding the `Model` as an additional parameter to the `NewMinimalVMI` method and 
* you should remove the optional description in the `... .To(` clause, because they do not differ from the description in the `It(...`.